include/spi_flash.h

```diff
@@ -233,8 +233,11 @@ static inline int spi_flash_erase(struct spi_flash *flash, u32 offset,
 static inline int spi_flash_protect(struct spi_flash *flash, u32 ofs, u32 len,
 					bool prot)
 {
-	if (!flash->flash_lock || !flash->flash_unlock)
-		return -EOPNOTSUPP;
+	if (!flash->flash_lock || !flash->flash_unlock) {
+		printf("*** Warning: flash lock/unlock ops not supported "
```

Comment:
so, without this patch, u-boot does not enter DFU mode; because `sf protect` returns `-EOPNOTSUPP`
the u-boot cmd that fails [if i remember correctly] is:
```
"jtagboot=env default -a;sf probe && sf protect unlock 0 100000 && run dfu_sf; \0"
```

another approach, would be to do a different u-boot for non-STMicro/Micron flashes without `sf protect` ; or to enter DFU mode regardless of the `sf protect` return code ;
atm, there is only 1 SidekiqZ2 board with an ISSI flash;
the other one [with STMicro/Micron] works without this change ;